lib/api/v3/work_packages/form/work_package_payload_representer.rb

```diff
@@ -64,6 +64,13 @@ def initialize(represented, options = {})
 
           property :lock_version
           property :subject, render_nil: true
+          property :done_ratio,
+                   as: :percentageDone,
+                   getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
+                   render_nil: false
+          property :estimated_hours,
```

Comment:
WTF?
As if this would cost time...

It ~~seems like~~ *is* a silly idea to implement it different from the WP representer. The effort should be something below an hour including tests. You only need to implement the parsing.